cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -312,9 +312,16 @@ export class Application {
             .click();
     }
 
+    filterTags(source: string): void {
+        // Filter tags on application details page
+        this.applicationDetailsTab("Tags");
+        cy.wait(2000);
+        cy.get("#source-filter-value-select").click();
+        cy.get("div.pf-c-select__menu").contains(source).click();
+    }
+
     tagAndCategoryExists(tags: string | string[][]): void {
         // Verify that tags and categories are present on Application details -> Tags page
-        this.applicationDetailsTab("Tags");
```

Comment:
Why are these two lines removed ?
I think it makes sense to navigate to the tab and then close it in the end .